cmd/jujud/agent/machine/manifolds.go

```diff
@@ -595,6 +647,18 @@ var ifNotMigrating = engine.Housing{
 	Occupy: migrationFortressName,
 }.Decorate
 
+var ifResponsibleController = engine.Housing{
+	Flags: []string{
+		isResponsibleControllerFlagName,
+	},
+}.Decorate
+
+var ifController = engine.Housing{
+	Flags: []string{
+		isControllerFlagName,
+	},
+}.Decorate
```

Comment:
While these seem cute and functional, and make for terse descriptions of the components to run, it feels a little like the contribute to the 'logging' spew when trying to figure out what is/should be/isn't running right now. Not running because "flag not set" is often not very understandable. Is it supposed to be set? is something currently running and going to be setting it soon?

Reply:
Yes, they are a bit noisy, and not completely clear when debugging. I don't have immediate ideas for improvement, will have to ponder.